flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/dialect/JDBCDialect.java

```diff
@@ -146,6 +153,42 @@ default String getSelectFromStatement(String tableName, String[] selectFields, S
 				.map(f -> quoteIdentifier(f) + "=?")
 				.collect(Collectors.joining(" AND "));
 		return "SELECT " + selectExpressions + " FROM " +
-				quoteIdentifier(tableName) + (conditionFields.length > 0 ? " WHERE " + fieldExpressions : "");
+			quoteIdentifier(tableName) + (conditionFields.length > 0 ? " WHERE " + fieldExpressions : "");
+	}
+
+	/**
+	 * Get create table statement.
+	 */
+	default String getCreateTableStatement(String tableName, TableSchema schema, String[] primaryKeyFields) {
+		List<String> expressions = new ArrayList<>();
+		for (TableColumn column : schema.getTableColumns()) {
+			expressions.add(quoteIdentifier(column.getName()) + " " + getDialectTypeName(column.getType()));
+		}
+		if (primaryKeyFields != null && primaryKeyFields.length > 0) {
+			String primaryKey = "PRIMARY KEY" + String.format("(%s)", StringUtils.join(
+				Arrays.stream(primaryKeyFields)
+					.map(name -> quoteIdentifier(name))
+					.collect(Collectors.toList()),
+				","));
+			expressions.add(primaryKey);
+		}
+
+		return String.format("CREATE TABLE IF NOT EXISTS %s (%s)", quoteIdentifier(tableName),
+			StringUtils.join(expressions, ","));
+	}
+
+	/**
+	 * Get dialect type name from {@link DataType}.
+	 */
+	default String getDialectTypeName(DataType dataType) {
```

Comment:
I don't think we should have a default type name mapping. Almost every database vendors have totally different type names. We should have the mapping clearly defined in dialect implementations.